lib/components/user/terms-of-use-pane.js

```diff
@@ -46,11 +50,19 @@ const TermsOfUsePane = ({
         >
           {/* TODO: Implement the link */}
             Optional: I consent to the Trip Planner storing my historical planned trips in order to
-            improve transit services in my area. <a href={`/#${TERMS_OF_STORAGE_PATH}`} target='_blank'>More info...</a>
+            improve transit services in my area. {termsOfStorageSet && <a href={`/#${TERMS_OF_STORAGE_PATH}`} target='_blank'>More info...</a>}
         </Checkbox>
       </FormGroup>
     </div>
   )
 }
+const mapStateToProps = (state, ownProps) => {
+  return {
+    otpConfig: state.otp.config
```

Comment:
Our general practice is to only supply the specific field to a component through mapStateToProps (rather than the whole config object).

Reply:
As I was writing it I knew something felt off... Thanks for catching that! Refactored in 930a4a7